_posts/Installation/0000-01-04-dev_install.md

```diff
@@ -1,38 +1,67 @@
+### Configuring the environment
+
+Set the needed variables in .evn (take librephotos.env as model). Make sure you set the `codedir` variable that tells docker where your source folders are. this should be the same path as used to clone your repos.
 
 Run `docker-compose -f docker-compose.yml -f docker-compose.dev.yml up -d`
 
-This will build images from scratch (can take a long time the first time)
-Now you can develop and benefit from code auto reload from both backend and frontend
-N.B. If you already built this image once, when you do force rebuild you have to run 
+This will build images from scratch (can take a long time the first time). Now you can develop and benefit from code auto reload from both backend and frontend
+
+If you already built this image once, when you do force rebuild you have to run the build command based on which area you changed with added dependencies/libraries etc.
+
+```bash
+docker-compose -f docker-compose.yml -f docker-compose.dev.yml  build --no-cache frontend
+# OR
+docker-compose -f docker-compose.yml -f docker-compose.dev.yml  build --no-cache backend
+```
 
-`docker-compose -f docker-compose.yml -f docker-compose.dev.yml  build --no-cache frontend`
+Once you have rebuilt, you can run the docker compose up command again to update the running instances.
 
-`docker-compose -f docker-compose.yml -f docker-compose.dev.yml  build --no-cache backend`
+```bash
+docker-compose -f docker-compose.yml -f docker-compose.dev.yml up -d
+```
 
-based on which one you changed if these changes need rebuild as for added dependencies/libraries etc.
+### Visual Studio Code
 
-and then the usual `docker-compose -f docker-compose.yml -f docker-compose.dev.yml up -d`
+If you use Visual Studio Code you can also benefit auto completion and other goodies. For this just type `code .` in your dockerfile folder which is `/dev/librephotos/librephotos-docker` using the example path above.
 
-If you use vscode you can also benefit auto completion and other goodies. For this just type `code .` in your dockerfile folder.
-Vscode will open and ask you if you want to reopen it in the container. If you do it he will add his server to the docker layers (first time you do it can take a couple of minutes) and you will have the same python interpreter librephotos is using internally hence the same installed libraries in auto completion and the same development environment will be shared by all devs (isort, flake8 and pylint for example)
\ No newline at end of file
+Visual Studio Code will open and ask you if you want to reopen it in the container. If you do it will create the images from scratch (first time you do it can take a couple of minutes) and you will have the same python interpreter librephotos is using internally hence the same installed libraries in auto completion and the same development environment will be shared by all devs including the tools like isort, flake8 and pylint.
```

Comment:
```suggestion
Visual Studio Code will open and ask you if you want to reopen it in the container. If you do it will create the images from scratch (first time you do it can take a couple of minutes) and you will have the same Python interpreter that LibrePhotos is using internally - and hence the same installed libraries in auto completion, and the same development environment, will be shared by all devs. This includes tools like isort, flake8 and pylint.

Alternatively, you can run the `Remote-Containers: Open Folder in Container` command from the Command Palette. Note that in order to open this container, you must have the required dependencies installed. More details can be found [here](https://code.visualstudio.com/docs/remote/containers).
```
Add more details about opening container in VS Code and link to documentation